manifests/b/BespokeSynth/BespokeSynth/1.1.0/BespokeSynth.BespokeSynth.locale.en-US.yaml

```
@@ -0,0 +1,67 @@
+Moniker: bespoke
+Tags: 
+- synthesizer
+- audio
+- music
+- DAW
+# Agreements: 
+ReleaseNotes: |
+  changes since 1.0.0:
+  · all new VST scanner/manager, and better support for large VST collections
+  · python is now packaged in with bespoke: no need to install a specific version
+  · bespoke is now MUCH easier to build from source. many thanks to @baconpaul and @mvf for this massive effort!
+  · added tooltips for spanish, german, and russian. many thanks to the translators!
+  · new modules: notetable, notetoggle, noteexpression
+  · added highlight when holding a patch cable over a target that can receive it
+  · added "throb" effect to patch cable bubbles when shift-dragging a module, to hint at autopatching functionality
+  · made effectchain modules more visible by adding them to the title bar dropdown
+  · improved settings menu: now most setting changes don't require a restart
+  · visual options: ability to disable background lissajous visualizer, set its color, set the canvas background color, and disable motion blur
+  · better sequencing options for controlsequencer and radiosequencer
+  · add ability to lock drumsequencer rows from being affected by randomizer
+  · fixed pop in drumsynth, and added oversampling option
+  · made it possible to launch bespoke by double-clicking a .BSK file
+  · updated large dropdown lists to stay on-screen, and get paging buttons if they're too large to fully display
+  · made it possible to spawn LFOs directly, rather than just via right-click on a slider
+  · fixed presets module, it didn't work at all in 1.0.0
+  · fixed bug where pulser didn't work in "free" mode
+  · fixed a bunch of little UI snags that I observed new users hitting
+  · microtonal support: the scale module can load SCL/KBM files, or follow oddsound
+  · VSTs now follow the transport correctly
+  · VST midi out support
+  · updated notefilter to not allow out-of-range pitches
+  · added hard limiter to output module, to prevent loud sounds from hitting your speaker
+  · added "always on top" setting for VST windows
+  · show module names in their title bar, for increased clarity
+  · added fix to improve linux performance and allow 60fps
+  · allow for more than 16 input/output channels
+  · fixed issue where presets didn't name modules correctly, which caused routing issues
+  · fixed issue where created modules would appear in the wrong location if you clicked quickly enough
+  · fixed issue where script modules with multiple note outputs wouldn't restore their savestate correctly
+  · fix VSTs with complex bussing being handled poorly by bespoke
+  · fixed windows hang caused by dynamic desktop wallpaper or unplugging/plugging in a display
+  · fixed various crashes
+  · added various optimizations
+  · other minor things
+ReleaseNotesUrl: https://github.com/BespokeSynth/BespokeSynth/releases
```

For a more permanent link it would be like this
```suggestion
ReleaseNotesUrl: https://github.com/BespokeSynth/BespokeSynth/releases/tag/v1.1.0
```